### PR TITLE
chg: dev: Modify pexpect logger to match underling data stream

### PR DESCRIPTION
**Pull request by jcrono**
This change adds a new filehandler that doesn't introduce line changes
arbitrarily. Also Add some line changes on known places to keep output
log file readable. With this changes the log file should match
pexpect's command output stream closer.

### CODE REVIEW

File: lib/topology/platforms/shell.py [hunk 1]
```diff
@@ -618,6 +618,7 @@ def connect(self, *args, connection=None, **kwargs):
                 ]),
                 category='connection'
             )
+        spawn.setwinsize(30, 150)
```
Comment (ocelotl) [inline lib/topology/platforms/shell.py]: Please explain why is this necessary.

File: lib/topology/logging.py [hunk 2]
```diff
@@ -45,6 +45,29 @@
 """Simple map with the default logging levels."""
 
 
+class FileHandler(logging.FileHandler):
```
Comment (ocelotl) [inline lib/topology/logging.py]: Is it strictly necessary for the name of this class to be the same as the one in ```logging```?

File: lib/topology/platforms/shell.py (hunk 1 above)
Comment (jcrono) [reply lib/topology/platforms/shell.py]: This helps minimize the amount of line changes inserted by the pseudo-tty pexpect uses

Comment (jcrono) [reply lib/topology/platforms/shell.py]: Want me to comment it in code?

File: lib/topology/logging.py (hunk 2 above)
Comment (jcrono) [reply lib/topology/logging.py]: Probably not, changing name

**Comment by jcrono** [comment]
Rquested changes made
